.github/workflows/CI.yml

```diff
@@ -79,6 +79,33 @@ jobs:
         with:
           files: lcov.info
 
+  thunderhead-ci:
+    name: Thunderhead CI Test using Tutorials- ${{ github.event_name }}
+    runs-on: thunderhead
+    environment: thunderhead-dev
```

Comment:
I don't think this is the intended use of `environment`. This script doesn't deploy anything, so shouldn't have an `environment`.

Reply:
I've had a look, and GitHub doesn't seem to have a way to scope secrets in another way. I think we should keep this, but add a comment on why we are using the environment in a non-standard way.